src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/forgotpassword/ForgotPasswordPage.java

```diff
@@ -0,0 +1,10 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormPage;
+
+public interface ForgotPasswordPage extends FormPage {
+
+  String pageTitle = "Forgot password";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)